setup.cfg

```diff
@@ -1,2 +1,51 @@
+[metadata]
+name = screed
+description = a Python library for loading FASTA and FASTQ sequences
+long_description = file: README.md
+long_description_content_type = text/markdown; charset=UTF-8
+url = https://github.com/dib-lab/screed
+author = Alex Nolley, C. Titus Brown
+author_email = ctbrown@ucdavis.edu,
+license = BSD 3-clause
+license_file = doc/LICENSE.rst
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Environment :: MacOS X
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Natural Language :: English
+    Operating System :: POSIX :: Linux
+    Operating System :: MacOS :: MacOS X
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Scientific/Engineering :: Bio-Informatics
+project_urls = 
+    Documentation = https://screed.readthedocs.io
+    Source = https://github.com/dib-lab/screed
+    Tracker = https://github.com/dib-lab/screed/issues
+
+[options]
+zip_safe = False
+packages = find:
+platforms = any
+include_package_data = True
+install_requires =
+    bz2file
+python_requires = >=3.7
+setup_requires =
+    setuptools_scm
+    pytest_runner
```

Comment:
I think we can drop the `pytest_runner` dependency, since it is [deprecated](https://pypi.org/project/pytest-runner/) and not recommended anymore?
```suggestion
```